front/lib/api/workspace.ts

```diff
@@ -109,6 +115,42 @@ export async function setInternalWorkspaceSegmentation(
   };
 }
 
+/**
+ * Get the white-listed providers for a workspace.
+ * @param workspaceId - The ID of the workspace.
+ * @returns A list of white-listed providers.
+ */
+export async function getWhiteListedProviders(
```

Comment:
I don't think we need / want this. That info is on the WorkspaceType, so we always have it

Reply:
Alright, thought that was good practice. Will ✂️ it 👍🏼